idx/idx.go

```diff
@@ -18,10 +18,27 @@ var (
 type Node struct {
 	Path        string
 	Leaf        bool
-	Defs        []schema.MetricDefinition
+	Defs        []Archive
 	HasChildren bool
 }
 
+type Archive struct {
```

Comment:
'archive' is the wrong name for this.  An 'archive' would be where the data is stored, this is just the index.